saml-core/src/main/java/org/keycloak/saml/common/util/SecurityActions.java

```diff
@@ -37,7 +37,7 @@
      *
      * @return
      */
-    static Class<?> loadClass(final Class<?> theClass, final String fullQualifiedName) {
+    public static Class<?> loadClass(final Class<?> theClass, final String fullQualifiedName) {
         SecurityManager sm = System.getSecurityManager();
 
```

Comment:
Each public static method call running privileged code should be guarded via checkPermission.
This applies to all methods in this class.